promptsource/templates/aeslc/templates.yaml

```
@@ -43,27 +47,31 @@ templates:
 
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: question_context_4
-    reference: ''
+    reference: Given a domain, ask a question
```

```suggestion
    reference: ''
```
what is "domain" here???